### ISSUE TITLE
Question: Possible to use without administrator privilege?

### ISSUE THREAD

**Issue by wimglenn** [glyph/MOPUp#385]
I've used MOPUp on my personal machine where I can enter admin password without issue. Works great.

Is it possible to use on a regular user account (installing to different prefix) without privilege? If so, could you please add instructions in the landing page, if not could you please mention that an administrator password is required.

Thanks!

**Comment by glyph** [comment]
Maybe!

I do not believe the official Python.org installers are relocatable, however.  For example, if I do `otool -L /Library/Frameworks/Python.framework/Python`, I don't see `@rpath` entries, just absolute paths.  This is why projects like https://github.com/gregneagle/relocatable-python exist.

I *vaguely* recall some interest from python.org in doing this, but I can't dig up any references now.  If this is possible, I'm happy to add to MOPUp to make it happen.

**Comment by wimglenn** [comment]
Yeah, seems like it's well out scope for MOPUp to attempt rpath rewriting and stuff. I'm using [indygreg/python-build-standalone](https://github.com/indygreg/python-build-standalone) in the meantime.

**Comment by glyph** [comment]
Maybe it would make sense for MOPUp to provide an option to install from there, if they're proper relocatable framework builds?

**Comment by wimglenn** [comment]
Personally, I don't think so. It would be reinventing what [`uv python install`](https://docs.astral.sh/uv/guides/install-python/#installing-a-specific-version) already does, except that MOPUp itself would need to find/get an existing Python runtime to use and `uv` doesn't.
